tests/bluetooth/tester/src/btp_zephyr.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I assume this file shouldn't have been pushed?

Reply:
removed